README.md

```diff
@@ -7,4 +7,8 @@ Travis:   [![Travis](https://travis-ci.org/aspnet/Antiforgery.svg?branch=dev)](h
 
 Antiforgery system for generating secure tokens to prevent Cross-Site Request Forgery attacks.
 
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+This project is part of ASP.NET Core. You can find documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+Samples can be found in [Entropy](https://github.com/aspnet/Entropy).
```

Comment:
I think we're trying not to tout them as "official" samples. They're just snippets of interest. The docs have official samples.